drools-game-engine-capture-flag-model/src/main/java/org/drools/game/capture/flag/model/Flag.java

```diff
@@ -48,4 +51,18 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public boolean equals(Object obj) {
+    	//TODO implement
+    	return this.name.equals(((Flag)obj).getName());
```

Comment:
This will not work as it is supposed to. I would rather let the IDE implement a tested equals and hashcode. And please add that to all the other model objects.